src/in-depth/exit-code.md

```diff
@@ -1 +1,55 @@
 # Exit codes
+
+A program doesn't always succeed.
+And when an error occurs,
+you should make sure to emit the necessary information correctly.
+In addition to
+[telling the user about errors](human-communication.md),
+on most systems,
+when a process exits,
+it also emits an exit code
+(an integer between 0 and 255).
```

Comment:
On POSIX, an exit code is a u8, while on Win32 an exit code is a u32 (weirdly, `std::process::exit` takes an i32, I'm not sure why). It's probably best not to mention the actual limits until you get to a "platform-specific behaviour" section.

Reply:
How about "an integer between 0 and 255 is compatible with most platforms"?